Vagrantfile

```diff
@@ -0,0 +1,23 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get update
+    # Install pyenv prerequisites
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+    # Install pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+    pyenv install 3.8.5
+    pyenv global 3.8.5
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+    cd /vagrant
+    poetry install
+    poetry run flask run

```

Comment:
```suggestion
    poetry run flask run
    poetry run flask run -h=0.0.0.0 -p=5000
```

Reply:
Note that the application should really be run in a trigger so that the vagrant up command can be allowed to finish:
```
  config.trigger.after :up do |trigger|
    trigger.name = "Launching App"
    trigger.info = "Running the TODO app setup script"
    trigger.run_remote = {privileged: false, inline: <<-SHELL
      cd /vagrant
      poetry install
      poetry run flask run -h=0.0.0.0 -p=5000
    SHELL
  }
```
Without this vagrant gets stuck in a bringing up state.